### PR TITLE
Update extension tutorial to use Manifest V3

### PR DESCRIPTION
**Pull request by eclipse439**
As requested by multiple users, this PR updates docs to change manifest version to v3, as well as replace any deprecated functions.

I've also added a PR to update the source code linked in the docs here: https://github.com/MicrosoftEdge/MicrosoftEdge-Extensions-Demos/pull/34

Issues requesting update: #2012 #1878 

**Rendered pages for review:**
https://review.docs.microsoft.com/en-us/microsoft-edge/extensions-chromium/getting-started/part1-simple-extension?branch=pr-en-us-2014
https://review.docs.microsoft.com/en-us/microsoft-edge/extensions-chromium/getting-started/part2-content-scripts?branch=pr-en-us-2014

### CODE REVIEW

**Comment by captainbrosset** [comment]
Thank you so much for the PR @eclipse439! We'll give this a review soon.

**Review by ashishnirvana1 — APPROVED** [review]
Iam fine by these changes, but am not fully sure if these cover everything. Will be nice to get a pass from Utkarsh Patankar as well.

**Comment by captainbrosset** [comment]
> Iam fine by these changes, but am not fully sure if these cover everything. Will be nice to get a pass from Utkarsh Patankar as well.

Sounds great to me. Are you able to at-mention them here? I don't know what their username is on GitHub.

**Review by Reezaali — CHANGES_REQUESTED** [review]
Still in review by @ashishnirvana1 to confirm that the tutorial is complete. We also need to implement tabs for MV2 and MV3. 

@mikehoffms  - can you update the doc with the tabs please?

**Comment by mikehoffms** [comment]
@Reezaali 
I implemented tabs to preserve Manifest V2 code listings.  The Files Changed tab served well, the sequence we used:

1. First, eclipse439 had a chance to view the individual changed lines in the **Files Changed** tab.

2. Then, after I added tabs, the **Files Changed** tab no longer shows individual changed lines, but rather, confirms my correct restoration of V2 code listings: it shows that this PR adds a tab demarcator above the V2 listings (no other changes to V2 listings), and adds whole complete new V3 listings (not depicted as individual line changes).

**Comment by Reezaali** [comment]
For now, I will leave my review as blocking the merge until we hear back from @ashishnirvana1 to confirm that no additional changes are needed to the tutorial. Thanks!

File: microsoft-edge/extensions-chromium/getting-started/part1-simple-extension.md [hunk 1]
```diff
@@ -23,7 +23,7 @@ The goal for this tutorial is to build a Microsoft Edge extension, starting with
 <!-- ====================================================================== -->
 ## Before you begin
 
-To test out the completed extension that you are building in this tutorial, download the [source code](https://github.com/MicrosoftEdge/MicrosoftEdge-Extensions-Demos/tree/master/extension-getting-started-part1/part1).<!-- changing master to main doesn't work 5/19/2022 -->
+To test out the completed extension that you are building in this tutorial, download the source code from the [MicrosoftEdge-Extensions-Demos repo > extension-getting-started-part1](https://github.com/MicrosoftEdge/MicrosoftEdge-Extensions-Demos/tree/master/extension-getting-started-part1/part1).<!-- changing master to main doesn't work 5/19/2022 -->  The source code has been updated from Manifest V2 to Manifest V3.
```
Comment (mikehoffms) [inline microsoft-edge/extensions-chromium/getting-started/part1-simple-extension.md:26]: @Reezaali - note that the Demos repo has a PR to replace V2 code in .json file by V3.

File: microsoft-edge/extensions-chromium/getting-started/part2-content-scripts.md [hunk 2]
```diff
@@ -5,11 +5,11 @@ author: MSEdgeTeam
 ms.author: msedgedevrel
 ms.topic: conceptual
 ms.prod: microsoft-edge
-ms.date: 01/07/2021
+ms.date: 06/16/2022
 ---
 # Create an extension tutorial, part 2
 
-[Completed extension package source for this part](https://github.com/MicrosoftEdge/MicrosoftEdge-Extensions-Demos/tree/master/extension-getting-started-part2/extension-getting-started-part2)<!-- changing master to main doesn't work 5/19/2022 -->
+To see the completed extension package source for this part of the tutorial, go to [MicrosoftEdge-Extensions-Demos repo > extension-getting-started-part2](https://github.com/MicrosoftEdge/MicrosoftEdge-Extensions-Demos/tree/master/extension-getting-started-part2/extension-getting-started-part2).<!-- changing master to main doesn't work 5/19/2022 -->   The source code has been updated from Manifest V2 to Manifest V3.
```
Comment (mikehoffms) [inline microsoft-edge/extensions-chromium/getting-started/part2-content-scripts.md:12]: re: part 2:
@Reezaali - note that the Demos repo has a PR to replace V2 code in .json file by V3.

**Comment by sonakshimore** [comment]
Utkarsh has suggested the below changes to the documentation:

1. Change suggested to [Create an extension tutorial, part 1](https://review.docs.microsoft.com/en-us/microsoft-edge/extensions-chromium/getting-started/part1-simple-extension?branch=pr-en-us-2014&tabs=v2#step-3-open-a-default-pop-up-dialog) documentation, Step #3 last sentence:

**Old version**: Finally, ensure you register the pop-up in `manifest.json` under `browser_action`, as shown in the following code snippet.
**New version**: Finally, ensure you register the pop-up in `manifest.json` under `browser_action` or `action` in Manifest V2 or Manifest V3, respectively, as shown in the following code snippet.

2. Extension demos on [Before you begin](https://review.docs.microsoft.com/en-us/microsoft-edge/extensions-chromium/getting-started/part1-simple-extension?branch=pr-en-us-2014&tabs=v3#before-you-begin) section of documentation:

Since it mentions that the source code has been updated from Manifest V2 to Manifest V3, we should actually update the source code first before we update this documentation.

Could you please review & approve [GitHub PR #34 - Update Edge Extension Tutorials to use Manifest V3 by eclipse439 · MicrosoftEdge/MicrosoftEdge-Extensions-Demos](https://nam06.safelinks.protection.outlook.com/?url=https%3A%2F%2Fgithub.com%2FMicrosoftEdge%2FMicrosoftEdge-Extensions-Demos%2Fpull%2F34%2Ffiles&data=05%7C01%7Cv-sonmore%40microsoft.com%7C12c53a58225446cb62fd08da55207eba%7C72f988bf86f141af91ab2d7cd011db47%7C1%7C0%7C637915895305363912%7CUnknown%7CTWFpbGZsb3d8eyJWIjoiMC4wLjAwMDAiLCJQIjoiV2luMzIiLCJBTiI6Ik1haWwiLCJXVCI6Mn0%3D%7C3000%7C%7C%7C&sdata=lVGQLtt8jHzwfKRpPehW%2FM0pVHZysHAf5wmYbTauaNk%3D&reserved=0)?

**Comment by Reezaali** [comment]
@mikehoffms - I merged PR34 above as mentioned by @sonakshimore. Thanks.

File: microsoft-edge/extensions-chromium/getting-started/part2-content-scripts.md [hunk 3]
```diff
@@ -112,7 +114,38 @@ if (sendMessageId) {
...
+        });
+    };
+}
+```
+
+---
+
+4. Make your `stars.jpeg` available from any browser tab
 
 You're probably wondering why, when you pass the `images/stars.jpeg` must you use the `chrome.extension.getURL` chrome Extension API instead of just passing in the relative URL without the extra prefix like in the previous section.  By the way, that extra prefix, returned by `getUrl` with the image attached looks something like the following.
```
Comment (mikehoffms) [inline microsoft-edge/extensions-chromium/getting-started/part2-content-scripts.md:150]: This V2-specific word|sentence|paragraph needs techreview in future PR.  chrome.extension.getURL is V2-specific.  @ashishnirvana1 @eclipse439

**Pull request merged**